pyproject.toml

```diff
@@ -8,7 +8,7 @@ version = "0.1.19"
 description = "All-in-one repository for state-of-the-art NeRFs"
 readme = "README.md"
 license = { text="Apache 2.0"}
-requires-python = ">=3.7.3"
+requires-python = ">=3.10"
```

Comment:
Needs to be reverted

Reply:
Yes! Sorry I missed that, thanks for letting me know. Will do tomorrow!